### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by Laxystem** [Gaming32/kilo-engine#4]
*This is mainly a list for myself, I'll update the README when I understand something*

- [x] Gradle Groovy Installation
- [x] Maven XML Installation
- [x] `level.json5`s
- [ ] Collision Types
- [ ] Namespaces (`namespace/levelname.scene.json5`)
- [ ] Entities
- [ ] PRP Shaders (Programmable Render Pipeline)

**Comment by Gaming32** [comment]
Namespaces?

**Comment by Gaming32** [comment]
Also most of this stuff should go in a wiki

**Comment by Laxystem** [comment]
> Also most of this stuff should go in a wiki

Oh sure, I'm just adding a list of stuff to learn/document. And I think it's better to use Javadocs than a wiki.

> namespaces?

Like, `example/example.level.json5`. the first `example` is the game/mod namespace (right?). I meant documentation of the asset tree by that.

**Comment by Gaming32** [comment]
There's not really any kind of namespace system or convention. It's just directories. Use them how you please.

**Comment by Gaming32** [comment]
Maybe stuff should go in the README, not a wiki, actually. Javadoc is better for API reference, while guides should generally go in a wiki or README.

**Comment by Laxystem** [comment]
> Maybe stuff should go in the README, not a wiki, actually. Javadoc is better for API reference, while guides should generally go in a wiki or README. 

Yeah, there's not enough contents for having a wiki. Plus, everyone can contribute to a README, but not to a wiki.
